### ISSUE TITLE
Suggested change for Icelandic TV schedules /endpoints/TV/index.js

### ISSUE THREAD

**Issue by HermannBjorgvin** [kristjanmik/apis#142]
I was playing around with the TV schedule API and noticed an improvement that could be made to the default response which lists the endpoints for each channel.

Basically if I want the Icelandic names of all the TV channels I have to program that into my application, doesn't sound to bad at first since this is relatively static data, but who wants to update their application each time a new TV channel pops up, that's what the API promises to do for you.

So here's my suggested change to the response, it wouldn't break preexisting applications and wouldn't require you to update your application if you wanted to get the Icelandic names of the channels as well

/endpoints/TV/index.js
```
 return res.json(
    {
      results: [
        {
          info: 'This is an api for Icelandic tv channel schedule\'s',
          endpoints: {
            ruv: '/tv/ruv/',
            ruvithrottir: '/tv/ruvithrottir/',
            stod2: '/tv/stod2/',
            stod2sport: '/tv/stod2sport',
            stod2sport2: '/tv/stod2sport2',
            stod2gull: '/tv/stod2gull',
            stod2bio: '/tv/stod2bio',
            stod3: '/tv/stod3',
            skjar1: '/tv/skjar1'
          },
          channels: {
            {name: 'Rúv', endpoint: '/tv/ruv/'},
            {name: 'Rúv íþróttir', endpoint: '/tv/ruvithrottir/'},
            {name: 'Stöð 2', endpoint: '/tv/stod2/'},
            {name: 'Stöð 2 Sport', endpoint: '/tv/stod2sport'},
            {name: 'Stöð 2 Sport 2', endpoint: '/tv/stod2sport2'},
            {name: 'Stöð 2 Gull', endpoint: '/tv/stod2gull'},
            {name: 'Stöð 2 Bíó', endpoint: '/tv/stod2bio'},
            {name: 'Stöð 2', endpoint: '/tv/stod3'},
            {name: 'Skjár 1', endpoint: '/tv/skjar1}'
          }
        }
      ]
    }
  );
```
Hope this or something like this gets added

**Comment by benediktvaldez** [comment]
I like it, not a breaking change so should be implementable right away..

@HermannBjorgvin could you make a pull request with these changes?

**Comment by HermannBjorgvin** [comment]
@benediktvaldez Done, if I did this correctly you should see it I think

**Comment by kristjanmik** [comment]
Merged in 4ac06668dc0aaf1bcabd0124f6f2a5bdd843c4eb
